src/qibolab/instruments/keysight_qcs.py

```diff
@@ -0,0 +1,201 @@
+"""Qibolab driver for Keysight QCS instrument set."""
+
+from typing import Dict, Tuple
```

Comment:
Just a very small simplification: we do not support any longer py3.8, so you could safely use `dict` and `tuple` (unless you need py3.8 explicitly, but then there would be other places where you may encounter errors)

Reply:
Fixed in 3fe629c5bbad306b58f9daec2fc155cca43b699f